entity-management-zoho/src/main/java/eu/europeana/entitymanagement/zoho/organization/ZohoOrganizationImporterConfiguration.java

```diff
@@ -0,0 +1,75 @@
+package eu.europeana.entitymanagement.zoho.organization;
+
+import java.io.File;
+import java.nio.file.Paths;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.context.annotation.PropertySources;
+
+import com.zoho.api.authenticator.store.FileStore;
+import com.zoho.api.authenticator.store.TokenStore;
+
+import eu.europeana.entitymanagement.common.config.AppConfigConstants;
+import eu.europeana.entitymanagement.zoho.ZohoAccessClient;
+
+@Configuration(AppConfigConstants.BEAN_ZOHO_ORGANIZATION_IMPORTER_CONFIGURATION)
+@PropertySource(value="classpath:zoho_import.properties", ignoreResourceNotFound = true)
+public class ZohoOrganizationImporterConfiguration {
```

Comment:
Please rename to ZohoAccessConfiguration and also rename the name of the bean

Reply:
Done.